content/articles/building-a-lyrics-search-app-using-vanilla-javascript/index.md

```diff
@@ -0,0 +1,490 @@
+
+## Building a Lyrics Search App Using Vanilla JavaScript
+
+![hero image](/building-lyrics-search-app-using-vanilla-javascript/hero.png)
+
+Not knowing the lyrics of a song, is a problem most song-lovers encounter. In this article, you will be creating a platform where users can search for lyrics by entering the artist name or title of the song.
+
+You will be creating the search app using HTML5, CSS3, Vanilla JavaScript, async-await with fetch method, OVH API, and ECMAScript 2015 (ES6) features like arrow functions.
```

Comment:
To this intro, add an overview of OVH API. Also, add how async and await would help and why are we using it here.

Reply:
If possible, rename this article with a better title.